src/spec/payid-protocol.md

```diff
@@ -449,12 +441,12 @@ This specification defines the PayID protocol - an application-layer protocol, w
 
   In this case, the PayID server MUST respond with the payment account information corresponding to at least one of the `[paymenNnetwork, environment]` tuples mentioned in the `Accept` header in the order of PayID client's preference. If none of those exist, then the PayID server MUST respond with payment account(s) information corresponding to all payment accounts associated with the queried PayID URI.
 
-# Versioning(TODO)
-
+# Versioning
  Versioning enables clients and servers to evolve independently. PayID protocol defines semantics for protocol versioning.
 
- PayID requests and responses are versioned according to the PayID-version header.
- PayID clients include the PayID-version header in order to specify the maximum acceptable response version. PayID servers respond with the maximum supported version that is less than or equal to the requested `major`
+ PayID requests and responses are versioned according to the PayID-Version header, as well as the `version` key in the PayID response.
+
+ PayID clients include the PayID-Version header in order to specify the maximum acceptable response version. PayID servers SHOULD respond with the maximum supported version that is less than or equal to the requested `major`.
```

Comment:
```suggestion
 PayID client requests SHOULD include the PayID-Version in any requests order to specify the maximum acceptable response version. PayID servers SHOULD respond with the maximum supported version that is less than or equal to the requested `major` version.
```

Reply:
IMO, PayID-Version header should be optional (MAY) and not a recommendation (SHOULD).